### PR TITLE
fn 0.4.143

### PR DESCRIPTION
**Pull request by lembacon**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/fn.rb [hunk 1]
```diff
@@ -27,7 +27,6 @@ def install
 
   test do
     require "socket"
-    assert_match version.to_s, shell_output("#{bin}/fn --version")
```
Comment (lembacon) [inline Formula/fn.rb]: Removed due to upstream forget to update their [hardcoded](https://github.com/fnproject/cli/blob/706f4031c48f9dc55b912a0845fe63bf1c170260/config/version.go#L13) version number.

Comment (DomT4) [reply Formula/fn.rb]: Probably worth `inreplace`-ing, or seeing if upstream want to fix it & cut a new release.

Comment (lembacon) [reply Formula/fn.rb]: `inreplace`-d, and reported upstream as fnproject/cli#376.

File: Formula/fn.rb [hunk 2]
```diff
@@ -15,6 +15,10 @@ class Fn < Formula
   depends_on "go" => :build
 
   def install
+    # Remove once fixed, due to upstream incorrectly hardcoded version number.
+    # See https://github.com/fnproject/cli/issues/376
+    inreplace "config/version.go", "0.4.135", version.to_s
```
Comment (DomT4) [inline Formula/fn.rb]: It might work fine without the `.to_s` up here, IIRC.

File: Formula/fn.rb (hunk 1 above)
Comment (javian) [reply Formula/fn.rb]: Strange that there's also a version.go file in the root directory which contains the correct version number.

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**